images/skopeo-ghcr-io.yaml

```diff
@@ -17,7 +17,8 @@ ghcr.io:
     cloudnative-pg/cloudnative-pg: ">= 1.22.2"
     dexidp/dex: ">=v2.40.0"
     external-secrets/external-secrets: ">= v0.7.0"
-    falcosecurity/rules/falco-rules: ">= 3.0.0"
+    falcosecurity/falco: ">= 0.39.2"
+    falcosecurity/falco-driver-loader: ">= 0.39.2"
```

Comment:
This one is [currently being retagged](https://github.com/giantswarm/retagger/blob/916248292b7a297f7b36ebed4c2fa44650e7d342/images/skopeo-docker-io.yaml#L125) from docker